statics/js/components/topology.js

```diff
@@ -154,6 +154,9 @@ var TopologyComponent = {
           <tab-pane title="Flows">\
             <flow-table-control></flow-table-control>\
           </tab-pane>\
+          <tab-pane title="Alert">\
+            <alert-form></alert-form>\
```

Comment:
could be really useful to have the list too

Reply:
done